03-transactions.md

```diff
@@ -13,7 +13,7 @@ Most transaction outputs used here are P2WSH outputs, the segwit version of P2SH
 ## Funding Transaction Output
 
 * The funding output script is a pay-to-witness-script-hash<sup>[BIP141](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#witness-program)</sup> to:
-   * `0 2 <key1> <key2> 2 OP_CHECKMULTISIG`
+   * `2 <key1> <key2> 2 OP_CHECKMULTISIG`
```

Comment:
Nice catch, it seems the zero was there in order to indicate an extra item on the stack to consume the extra pop as a result of the CMS bug within Script. The extra (dummy) item appears on the _witness_, not the witness script.